### PR TITLE
Adapt tensorflow and pytorch to new transform.py

### PR DESCRIPTION
**Pull request by marxprop**

### CODE REVIEW

**Review by arnauddgbes — APPROVED** [review]

**Review by arnauddgbes — APPROVED** [review]

**Review by arnauddgbes — APPROVED** [review]

File: dgbpy/uikeras.py [hunk 1]
```diff
@@ -106,16 +108,26 @@ def getAdvancedUiPars(uipars=None):
   dict = keras_dict
   uiobjs={}
   if not uipars:
+    aug_labels = ['Random Flip', 'Random Gaussian Noise']
+    if hasOpenCV(): aug_labels.append('Random Rotation')
     uiobjs = {
-      'augmentfld': CheckboxGroup(labels=['Enable Data Augmentation'], visible=True, margin=(5, 5, 0, 5)),
+      'scalingheadfld' :Div(text="""<strong>Data Scaling</strong>""", height = 10),
+      'scalingfld': RadioGroup(labels=['Global Normalization', 'Local Standardization', 'Local Normalization', 'MinMax Scaling'],
```
Comment (arnauddgbes) [inline dgbpy/uikeras.py]: not Global Normalization, Global Standardization

Use the keys defined in dgbpy.keystr.py

**Review by arnauddgbes — CHANGES_REQUESTED** [review]

**Review by arnauddgbes — CHANGES_REQUESTED** [review]
Same, use the keys defined in dgbpy/keystr.py

**Review by arnauddgbes — APPROVED** [review]

**Review by arnauddgbes — APPROVED** [review]
